experimental/dds/tree2/src/simple-tree/editNode.ts

```diff
@@ -53,7 +53,7 @@ export function getEditNode(target: TreeNode): FlexTreeNode {
 /**
  * Retrieves the edit node associated with the given target via {@link setEditNode}, if any.
```

Comment:
Do the docs need to be updated? Or are they still correct despite the rename?

Reply:
I.e. is "edit node" still the right terminology?

Reply:
"editNode" is a short hand invented in this folder to refer to the editiable-tree-2's TreeNode (now called FlexTreeNode) associated with a simple-tree TreeNode. There are quite a few places (like this file's name) that should be changed. I avoided renaming stuff I wasn't already messing with in this change. I'll do a more complete rename as a sperate PR.